### DISCUSSION TITLE
I don't know my password.

### DISCUSSION THREAD

**Discussion by wincol1151** [nextcloud/all-in-one#3563] [Questions] [answered]
I installed Nextcloud but when I try to log in for the first time, it asks me for a password. However, I don't know what the password is.
。

**Comment by szaimen** [comment, marked as answer]
Hi, did you saw this?
![image](https://github.com/nextcloud/all-in-one/assets/42591237/25947c81-596d-4767-8ff1-35f05ba071fd)

**Comment by wincol1151** [comment]
Ok, it has been solved perfectly. I deleted the volume nextcloud-aio-mastercontainer and was able to reset the password with a new one, then successfully logged in.

**Comment by wincol1151** [comment]
I saw the password field before, but I clicked next immediately and never had the chance to see the password again. This is why I can't log in without the password.

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/discussions/1786 then

**Comment by wincol1151** [comment]
I'm stuck on the reverse proxy issue now. I haven't been able to successfully set up the reverse proxy on my Synology NAS, so I can't log in to Apache's 11000 port. I'll try setting up the reverse proxy on the Synology NAS again, but if it doesn't work, I'll build a Nginx server instead.

**Reply by szaimen** [reply]
What kind of issue?
